apps/judicial-system/web/src/routes/Prosecutor/InvestigationRequest/PoliceReport/PoliceReportForm.tsx

```diff
@@ -13,58 +13,38 @@ import {
 } from '@island.is/judicial-system-web/src/utils/formHelper'
 import { useCase } from '@island.is/judicial-system-web/src/utils/hooks'
 import { icReportForm } from '@island.is/judicial-system-web/messages'
-import {
-  FormSettings,
-  useCaseFormHelper,
-} from '@island.is/judicial-system-web/src/utils/useFormHelper'
+import { isPoliceReportStepValidIC } from '@island.is/judicial-system-web/src/utils/validate'
 import type { Case } from '@island.is/judicial-system/types'
 import * as Constants from '@island.is/judicial-system-web/src/utils/constants'
 
 interface Props {
   workingCase: Case
   setWorkingCase: React.Dispatch<React.SetStateAction<Case>>
   isLoading: boolean
-  isCaseUpToDate: boolean
 }
 
 const PoliceReportForm: React.FC<Props> = (props) => {
-  const { workingCase, setWorkingCase, isLoading, isCaseUpToDate } = props
-  const validations: FormSettings = {
-    caseFacts: {
-      validations: ['empty'],
-    },
-    legalArguments: {
-      validations: ['empty'],
-    },
-  }
+  const { workingCase, setWorkingCase, isLoading } = props
+
   const { formatMessage } = useIntl()
   const { updateCase, autofill } = useCase()
+
   const [caseFactsEM, setCaseFactsEM] = useState<string>('')
   const [legalArgumentsEM, setLegalArgumentsEM] = useState<string>('')
-  const { isValid } = useCaseFormHelper(
-    workingCase,
-    setWorkingCase,
-    validations,
-  )
-  const defaultProsecutorOnlySessionRequest = formatMessage(
-    icReportForm.prosecutorOnly.input.defaultValue,
-  )
+
   useEffect(() => {
-    if (isCaseUpToDate) {
```

Comment:
We don't need to check for `isCaseUpToDate` here because the autofill happens when the user checks the "requestProsecutorOnlySession (Beiðni um dómþing að varnaraðila fjarstöddum)" checkbox, not on render

Reply:
Interesting, this is a bug then on staging.